### ISSUE TITLE
Add command line args to settings file

### ISSUE THREAD

**Issue by bendnorman** [catalyst-cooperative/pudl#1289]
The prefect branch introduces a lot of new command line args for deploying on the cloud. Add command line args to the new settings file.

**Comment by zaneselvans** [comment]
@bendnorman I'm wondering if we can should close this and also the #909 Epic, with all the settings stuff that's been done now? Or should we wait on the prefect merge discussion #1406

**Comment by bendnorman** [comment]
Good questions. I think it makes sense to close this and the #909 Epic. I'm not sure if it makes sense to move our command line args to our settings file.  

How we handle our command line args are dependent on which orchestration tool we go with.

**Comment by zaneselvans** [comment]
No longer relevant due to the Dagster refactor. See e.g. #1960 #2104
